CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.0
+
+The is a major release
+
+* [TK-369](https://tickets.puppetlabs.com/browse/TK-369) Move Jetty dependency to 9.4.1
```

Comment:
I don't think there are any user facing changes, so I'm not sure what else to put into the changelog here.